index.html

```diff
@@ -11,14 +11,79 @@
     />
   </head>
   <body class="app-grid">
-    <section class="header"></section>
 
-    <section class="content"></section>
+    <section class="header header__bar">
+      <p>Quiz App</p>
+    </section>
+
+    <section class="content">
+<div class="content__area">
+  
+
+    <div class="content__container container--scroll">
```

Comment:
This should be a section and is a new starting point for BEM. 
```suggestion
    <section class="quiz-card">
```